field.storage.config_pages.su_site_type.yml

```diff
@@ -0,0 +1,55 @@
+uuid: aef7044a-cf89-41f3-b683-aff011270c5b
```

Comment:
This file should be in the `config/sync` directory

Reply:
*facepalm* the two files are moved.